sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/spanner/MutationSizeEstimatorTest.java

```diff
@@ -81,10 +100,16 @@ public void nullPrimitiveArrays() throws Exception {
             .build();
     Mutation bool =
         Mutation.newInsertOrUpdateBuilder("test").set("one").toBoolArray((boolean[]) null).build();
+    Mutation numeric =
+        Mutation.newInsertOrUpdateBuilder("test")
+            .set("one")
```

Comment:
Also add tests for reading/writing the new type ?

Reply:
Writing NUMERIC was added in #12818.
Reading is automatically handled by spanner client libraries.